students/JackieCao/Lesson02/series.py

```diff
@@ -0,0 +1,76 @@
+# Fibonacci Series
+
+def fibonacci(n):
+"""
+make sure the input n is an integer and larger or equal to 0
+"""
+    if n != int(n):
+        if not isinstance(n,int):
+ 	    print("Hey, there's an error, n must be integer.")
+    elif n < 0:
+	print("Hey, there's an error, n must be larger or equal to 0.")
+    else:
+	n = int(n)
+	if n >= 2:
+	    """
+ 	    make an empty list has length n+1, n start from 0 
+	    the first two values are 0 and 1 for the Fibonacci series
+	    """
+	    L = [None]*(n+1)
+            L[0] = 0 
+	    L[1] = 1
+            for i in range(2,n+1):
+		"""
+	        this equation is based on the Fibonacci definition
+		"""
+	        L[i] = L[i-2]+L[i-1] 
+	    print(L[n]) #print the output
+        if n == 0:
+	    print(0)	    
+        if n == 1:
+	    print(1)	    
```

Comment:
Make sure your indents are consistent - this is very hard to run.  Seems to work well, though passing in 1 or 0 doesn't return anything.